src/App.tsx

```diff
@@ -50,84 +50,112 @@ export default function App(): JSX.Element {
     <div className="App">
       <AuthContext.Provider value={{ authTokens, clearAuth }}>
         <Router>
-          <NavigationAndFooterWrapper>
-            <Main id="main">
-              <Switch>
-                <Route exact path="/">
+          <Switch>
+            <Route exact path="/">
+              <NavigationAndFooterWrapper renderFooter>
```

Comment:
`<NavigationAndFooterWrapper renderFooter>` -> `<PageWithNavigation withFooter>`?

Reply:
Hmm, now when I rethink, my suggestion is not the the greatest. But I was thinking about the name now when the footer is optional.

Reply:
True it is little confusing, since it won't always render the Footer. Would it be better to just render Navigation only for the form pages? Think I tried something like that before during this story but there was some problem with it. I could try it again?

Reply:
Yes, maybe it would more clear with that one 🤔

Reply:
But if it gets too complicated, we can go with the current solution too 👍

Reply:
- [ ] Render only Navigation component to form pages instead of NavigationAndFooter wrapper